Project1/project1/.env

```diff
@@ -0,0 +1,3 @@
+FLASK_APP=application.py
+DATABASE_URL=postgres://nesrqsgxfpkmhs:837e8074d7ce3afe50e5e7ced003d7e2a9440c3a92647793a2225179fe0d847e@ec2-54-147-209-121.compute-1.amazonaws.com:5432/desi636rresb9m
```

Comment:
remove the url, it needs to be given from cmd for security reasons

Reply:
Data removed